src/gitMenuCommands.ts

```diff
@@ -103,4 +110,39 @@ export function addCommands(
       settings.set('simpleStaging', !settings.composite['simpleStaging']);
     }
   });
+
+  /** Command to add a remote Git repository */
+  commands.addCommand(CommandIDs.gitAddRemote, {
+    label: 'Add remote repository',
+    caption: 'Add a Git remote repository',
+    isEnabled: () => model.pathRepository !== null,
+    execute: async args => {
+      if (model.pathRepository === null) {
+        console.warn('Not in a Git repository. Unable to add a remote.');
+        return;
+      }
+      let url = args['url'] as string;
+      let name = (args['name'] as string) || 'origin';
```

Comment:
`Origin` is a sensible default to have, but since the server is already setting it, can we skip it at the frontend to avoid duplication?

Reply:
I agree